content/en/blog/_posts/2025-05-01-image-volume-beta/index.md

```diff
@@ -0,0 +1,108 @@
+---
+layout: blog
+title: "Kubernetes 1.33: Image Volumes graduate to beta!"
+date: 2025-05-01
+draft: true
+slug: image-voume-beta
```

Comment:
```suggestion
slug: kubernetes-1-33-image-volume-beta
```

This may need "v" prefix -- there is a separate discussion on this, and I'll make sure this gets handled.